platform/darwin/src/MGLAccountManager.h

```diff
@@ -1,11 +1,12 @@
-#import <Foundation/Foundation.h>
+#import "MGLTypes.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 /**
  The MGLAccountManager object provides a global way to set a Mapbox API access
  token.
  */
+MGL_EXTERN
```

Comment:
@frederoni I cannot confirm that behavior; it works for me with the macro as well.